releases/release-1.29/release-notes/release-notes-draft.json

```diff
@@ -678,8 +678,8 @@
   },
   "118619": {
     "commit": "19deb04a9079e01e0f9e04cd8f94a1173694916b",
-    "text": "Dynamic resource allocation: avoid creating a new gRPC connection for every call of prepare/unprepare resource(s).",
-    "markdown": "Dynamic resource allocation: avoid creating a new gRPC connection for every call of prepare/unprepare resource(s). ([#118619](https://github.com/kubernetes/kubernetes/pull/118619), [@TommyStarK](https://github.com/TommyStarK))",
+    "text": "Dynamic resource allocation: avoided creating a new gRPC connection for every call of prepare/unprepare resource(s).",
```

Comment:
for the release notes of updates that are adding features for the future, 'will now' should have been used instead